PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -19,6 +19,19 @@
   <change-notes>
     <![CDATA[
     <html>
+      <h3>[3.12.0]</h3>
+      <ul>
+        <li>Updated to list both Windows and Linux web apps in web apps explorer.</li>
+        <li>Added support to choose a Linux web app to deploy the application.</li>
+        <li>Added support to create a new Linux web app to deploy the application.</li>
+        <li>Added support to show all the deployment slots for a web app.</li>
+        <li>Refactored web apps icons.</li>
+        <li>Refactored the use existing web apps UI when users click 'Run on Web App'.</li>
```

Comment:
I suggest to remove this line. Or if you do want to mention the UI change, I think it would be "Refactor the UI for xxxx dialog."